oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/record-requests/components/RequestList.jsx

```diff
@@ -36,16 +36,17 @@ export const RequestList = ({ requests, requestTypes, requestModalType }) => {
           <RequestModal key={request.id} request={request} requestTypes={requestTypes} requestModalType={modalType}
             triggerButton={
               <List.Item as="a" key={request.id}>
-                <List.Content floated="right" verticalAlign="middle">
+                <List.Content floated="right" verticalAlign="middle" style={{ fontSize: "0.8rem" }}>
                   <div style={{ textAlign: "right" }}>{request?.status ?? i18next.t("No status")}</div>
                   {request?.created && <div>{request.created}</div>}
                 </List.Content>
-                <List.Content>
-                  <List.Header>{!_isEmpty(request?.title) ? request.title : (!_isEmpty(request?.name) ? request.name : request.type)}</List.Header>
-                  <List.Description>{_truncate(request.description, { length: 30 })}</List.Description>
+                <List.Content style={{ lineHeight: "1.3rem" }}>
+                  <List.Header style={{ fontWeight: 500 }}>{!_isEmpty(request?.title) ? request.title : (!_isEmpty(request?.name) ? request.name : request.type)}</List.Header>
```

Comment:
Avoid inlined style, noone can reuse & customize that (and we are in public library here)